### ISSUE TITLE
alternativne renderovanie cyklotras

### ISSUE THREAD

**Issue by jose1711** [FreemapSlovakia/FreemapDiSK#80]
bolo by fajn mat alternativnu vrstvu pre cyklotrasy v pripadoch, ze mam na freemape zapnutu (v expert rezime) sucasne vrstvu s turistickymi i cyklotrasami - tym padom by som vedel taketo trasy vizualne rychlo rozpoznat. casto byvaju cyklotrasy vyznacovane "gulickami" alebo hrubsou prerusovanou ciarou.

**Comment by seg-on** [comment]
Som sa raz hral s Maperitiv-om a spravil som takéto riešenie
![image](https://user-images.githubusercontent.com/19683952/48227888-eec71900-e3a3-11e8-86ae-b0f92d120028.png)

**Comment by seg-on** [comment]
je to kombinácia dvoch označení. bodiek pozdĺlž celej trasy vedených s miernym odstupom a občasnej ikonky bicyklu.
turisticé značené trasy sa prekrývajú s cestu a tak bodky cyklotrasy by malo byť stále vidno.

**Comment by jose1711** [comment]
jo, take by kludne mohlo byt. otazka je, ci by sa to vedel "naucit" aj disk.

**Comment by seg-on** [comment]
To neviem :/
